openapi/openapi.yml

```diff
@@ -0,0 +1,246 @@
+openapi: 3.0.3
+info:
+  title: Jellyfish REST API - OpenAPI 3.0
+  description: REST API of Jellyfish Media Server
+
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.1.0
+
+servers:
+  - url: https://localhost/api/v1
+tags:
+  - name: room
+    description: Manage rooms
+paths:
+  /room:
```

Comment:
Just a thought - with Jellyfish being a high-level API maybe we should consider some other name? Especially for scenarios with an HLS broadcast of a few (or one) presenters.

Reply:
I thought the same but nothing better came to my mind. Thought about: cluster, chain, container

Reply:
IMO, room is the best option at the moment